src/mixins/common-mixin.ts

```diff
@@ -49,13 +49,13 @@ export default class CommonMixin extends Vue {
     return fullName.trimRight()
   }
 
-  /** Return true when in correction filings */
+  /** Return true when in correction filings. */
   isCorrection (): boolean {
-    return this.$route.name === RouteNames.CORRECTION
+    return (this.$route.name === RouteNames.CORRECTION)
```

Comment:
Just curious, what's the difference here?

Reply:
I think brackets around such expressions are easier to understand at a glance (and possibly understand altogether if the expression gets more complicated with ternary operators, ands, ors, etc), so I've made this part of my code style and I'm applying it here because I was in this area of code.

(As for how I understand this: the "return" statement returns a single value, so I like to see a "single" thing -- something in brackets -- there. If "returns" returns just a variable then obviously brackets are not necessary for understanding.)

Reply:
fair enough!